test/performance/fixtures/perf-utils.ts

```diff
@@ -133,6 +133,18 @@ export class PerfUtils {
 			() => window?.wp?.blocks && window?.wp?.data
 		);
 
+		const initialRenderingMode = await this.page.evaluate( () => {
+			const { select } = window.wp.data;
+			return select( 'core/editor' ).getRenderingMode();
+		} );
+
+		if ( initialRenderingMode !== 'post-only' ) {
+			await this.page.evaluate( () => {
+				const { dispatch } = window.wp.data;
+				dispatch( 'core/editor' ).setRenderingMode( 'post-only' );
+			} );
+		}
```

Comment:
I actually wonder if this should be in this function or directly in the tests that actually load pages. Because it's not clear why a "load blocks" function should change the rendering mode.

Reply:
Fixed in [9068ab1](https://github.com/WordPress/gutenberg/pull/62304/commits/9068ab10786095bd2050e51839d5d5022b887616)